File: mpp_proiect_java/TriatlonModel/src/main/java/domain/Arbitru.java
package domain;

import java.io.Serializable;

public class Arbitru extends Entity<Long> implements Serializable {

    private String firstName;
    private String lastName;
    private String email;
    private String userName;
    private String password;
    private long responsabil_proba;
    private long id;

    public Arbitru(Long idArbitru,String firstName, String lastName,String email,String password,String userName,long responsabil_proba){
        setId(idArbitru);
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.userName = userName;
        this.password = password;
        this.responsabil_proba = responsabil_proba;
    }
    public Arbitru(){

    }


    public Long getid() {
        return super.getId();
    }

    public void setid(Long id){
        super.setId(id);
    }

    @Override
    public String toString() {
        return "Arbitru{" +
                "firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                '}';
    }
    public long getIdProbaArbitru(){
        return responsabil_proba;
    }


    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }


    public long getResponsabil_proba() {
        return responsabil_proba;
    }

    public void setResponsabil_proba(long responsabil_proba) {
        this.responsabil_proba = responsabil_proba;
    }
}

File: mpp_proiect_CSharp/triatlon/model/Arbitru.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace triatlon.domain
{
    [Serializable]
    public class Arbitru : Entity<long>
    {
        public long Id { get; set; }

        public string firstName { get; set; }

        public string lastName { get; set; }

        public string email { get; set; }

        public string username { get; set; }

        public string password { get; set; }

        public long id_proba_responsabil { get; set; }
  
        public Arbitru(long id, string firstname, string lastname, string email1, string username1, string password1,long id_proba_responsabil1)
        {
            this.Id = id;
            this.firstName = firstname;
            this.lastName = lastname;
            this.email = email1;
            this.username = username1;
            this.password = password1;
            this.id_proba_responsabil = id_proba_responsabil1;

        }
        public Arbitru(string username1, string password1)
        {
            this.username = username1;
            this.password = password1;
        }
        public override string ToString()
        {
            return "Arbitru - nume: " + firstName + '|' +
                "prenume=" + lastName;
        }

    }
}
